features/fixtures/mazeracer/app/src/main/java/com/bugsnag/mazeracer/scenarios/FixedSamplingProbabilityScenario.kt

```diff
@@ -0,0 +1,55 @@
+package com.bugsnag.mazeracer.scenarios
+
+import android.util.Log
+import com.bugsnag.android.performance.BugsnagPerformance
+import com.bugsnag.android.performance.PerformanceConfiguration
+import com.bugsnag.android.performance.internal.InternalDebug
+import com.bugsnag.mazeracer.Scenario
+
+class FixedSamplingProbability1Scenario(
+    config: PerformanceConfiguration,
+    scenarioMetadata: String
+) : Scenario(config, scenarioMetadata) {
+
+    var spanId = 1
+
+    init {
+        config.samplingProbability = 1.0
```

Comment:
These `Scenario` classes could be rolled into the same class:
```suggestion
        config.samplingProbability = scenarioMetadata.toDoubleOrNull() ?: 1.0
```